.golangci.yml

```diff
@@ -1,27 +1,24 @@
 linters:
   enable:
-    - deadcode
```

Comment:
deadcode, structcheck and varcheck linters are warned as removed by the linter, so I removed them from here.